src/rpdk/core/contract/suite/handler_commons.py

```diff
@@ -50,8 +50,7 @@ def test_create_failure_if_repeat_writeable_id(resource_client, current_resource
 @response_contains_resource_model_equal_current_model
 def test_read_success(resource_client, current_resource_model):
     primay_identifier_only_model = create_model_with_properties_in_path(
-        current_resource_model.copy(),
-        resource_client.primary_identifier_paths,
+        current_resource_model.copy(), resource_client.primary_identifier_paths,
```

Comment:
You need to update your black python module and may have to run the files individually. When the build was failing for me, these files weren't getting changed when I ran pre-commit for all files and needed to specifically run black on these specific files (though not sure if you'll have that same issue). It's definitely annoying that these linter/formatting will randomly fail if there was an update and things changed. Not sure if there's a way to have some configuration that will prevent updates to the linter/formatter causing changes to code we haven't touched

Reply:
+1, hit unrelated linting failures in https://github.com/aws-cloudformation/cloudformation-cli/pull/554 and seems weird to add unrelated formatting changes to these PRs